4. User Guides/Home Screen/README.md

```diff
@@ -1,6 +1,7 @@
 # The Home screen
 
-The first time you connect and login to a server you will be show the Rocket.Chat home screen.
+The first time you connect and login to a server you will be shown the Rocket.Chat home screen.
+Go to APP SETTINGS -> Layout to customize this intro.
```

Comment:
I think that this line should not be here, since this is the `User Guides` section, it should belong to the 'Administration' section